### PR TITLE
fix(list) : ellipsis on expanded display

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

On the expanded view, there is no ellipsis on elements

**What is the new behavior?**

Now there is ellipsis 

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

File: packages/components/src/List/DisplayLarge/DisplayLarge.scss [hunk 1]
```diff
@@ -40,6 +40,13 @@ $tc-list-display-large-background-color: lighten($tc-list-display-large-color, 2
 
 	& ul {
 		flex: 1;
+		overflow: hidden;
+		& li {
```
Comment (frassinier) [inline packages/components/src/List/DisplayLarge/DisplayLarge.scss]: `&` is useless

**Comment by frassinier** [comment]
The example does not represent ellipsis, maybe you should fix author max-width in Storybook?

File: packages/components/src/List/DisplayLarge/DisplayLarge.scss (hunk 1 above)
Comment (talend-sromain) [inline packages/components/src/List/DisplayLarge/DisplayLarge.scss]: as the ul just before i guess ?

**Comment by talend-sromain** [comment]
i see the ellipsis in this case : 
http://241.talend.surge.sh/components/?selectedKind=List&selectedStory=Large&full=0&down=1&left=1&panelRight=0&downPanel=kadirahq%2Fstorybook-addon-actions%2Factions-panel

@frassinier don't you have one here in the third row ?
If not, sure, i can update this example

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

File: output/components.sasslint.txt [hunk 2]
```diff
@@ -20,6 +20,9 @@ src/Drawer/Drawer.scss
   46:2   error    Class '.drawerStacked' should be written in lowercase with hyphens        class-name-format
   54:1   warning  Space expected between blocks                                             empty-line-between-blocks
 
+src/List/DisplayLarge/DisplayLarge.scss
```
Comment (jlame) [inline output/components.sasslint.txt]: lint issue

Comment (jlame) [inline output/components.sasslint.txt]: otherwise lgtm

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://241.talend.surge.sh)

**Pull request merged**